p2p/libp2p/networksharding/noSharder.go

```diff
@@ -3,6 +3,7 @@ package networksharding
 import (
```

Comment:
This also can be confused with `disabledSharder`, maybe this can be renamed also

Reply:
refactored naming